### PR TITLE
Update ua.h

### PR DESCRIPTION
**Pull request by v2e**
Updated to the current English version.

### CODE REVIEW

File: langs/ua.h [hunk 1]
```diff
@@ -40,12 +40,30 @@ msgstr("Помилка: Невірний Tox ID (антиспам)")
...
+msgstr("Позначте область екрана для надсилання.")
+
 msgid(TRANSFER_NEW)
 msgstr("Передача файлу")
 
 msgid(TRANSFER_STARTED)
 msgstr("Розпочато передачу файлу")
 
+msgid(TRANSFER___)
+msgstr("...")
```
Comment (tsudoko) [inline langs/ua.h]: Same as in English? You don't need to copy those over.

File: langs/ua.h [hunk 2]
```diff
@@ -265,11 +325,41 @@ msgstr("Заглушити")
...
+msgstr("Темне")
+
+msgid(THEME_HIGHCONTRAST)
+msgstr("Контрастне")
+
+msgid(THEME_CUSTOM)
+msgstr("Власне (див. докум.)")
+
+msgid(THEME_ZENBURN)
+msgstr("Zenburn")
```
Comment (tsudoko) [inline langs/ua.h]: same as above

**Comment by GrayHatter** [comment]
If you can delete the lines @tsudoko commented on I'll merge this

**Comment by v2e** [comment]
To be honest, I do not completely understand why is it necessary to remove them.
From my point of view,
* First of all, it is much easier to compare two files when they correspond line by line to each other. This way it is much faster to update the translation in the future.
* Second, somebody else may think that the omitted lines are new in the future, and try to translate them. On the contrary, when they are copied over, the future translators will see that those strings were copied on purpose.
* Third, if the lines are not for translation, why making them "translatable" at all? Maybe, they should be hardcoded in such case?
* Fourth, what is the actual reason to omit as many lines as possible and sacrifice the line-by-line correspondence between the files (seems important to me)? Is it saving memory?

**Comment by tsudoko** [comment]
1,2: Comment them out
3: Some languages might have different rules/conventions than others, so those "untranslatable" strings might get changed in their translation. I think there are no such languages in uTox right now, though.

**Comment by v2e** [comment]
So what is the final solution? Should I keep those strings, but comment them out?

**Comment by GrayHatter** [comment]
yeah do that. If the translation is the same there's no reason to have to have to change it in multiple files

**Comment by v2e** [comment]
I commented those lines out, like you asked.

**Comment by GrayHatter** [comment]
thank you sir!

**Pull request merged**